include/nuttx/fs/fs.h

```diff
@@ -300,6 +302,7 @@ struct mountpt_operations
   off_t   (*seek)(FAR struct file *filep, off_t offset, int whence);
   int     (*ioctl)(FAR struct file *filep, int cmd, unsigned long arg);
   int     (*truncate)(FAR struct file *filep, off_t length);
+  int     (*mmap)(FAR struct file *filep, FAR struct mm_map_entry_s *map);
```

Comment:
let's move mmap before truncate? Since mtd_dev_s and block_operations may add mmap, but without truncate.

Reply:
good idea, I'll do that as well, but tomorrow, I am in the middle of something else atm.